src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.vcxproj

```diff
@@ -27,6 +27,15 @@
   <Import Project="$(SolutionDir)\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
   <Import Project="$(OpenConsoleDir)\src\cppwinrt.build.pre.props" />
 
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!-- For CLI11: It uses dynamic_cast to cast types around, which depends
+      on being compiled with RTTI (/GR). I don't know what kind of hit this has
+      on us. -->
+      <RuntimeTypeInfo>true</RuntimeTypeInfo>

```

Comment:
Broadly -- is it possible to ship CLI11 as a static lib and only enable RTTI for that one lib? It would reduce the impact on us greatly.

Reply:
Or, build it into a static lib and then consume it later. not "ship it as one"

Reply:
_oh it is a hpp file ignore me_